packages/react-vanilla-components/src/components/ReCaptcha.tsx

```diff
@@ -30,13 +31,16 @@ const ReCAPTCHAComponent = (props: PROPS) => {
     appliedCssClassNames, isError, errorMessage, dispatchChange
   } = props;
   const config = getOrElse(props, ['properties', 'fd:captcha', 'config'], {});
-  const bemClass = 'cmp-adaptiveform-recaptcha';
-
+  const isHCaptcha = function() {
+    return (props[':type'].includes('hcaptcha'));
+  };
+  const bemClass = isHCaptcha() ? 'cmp-adaptiveform-hcaptcha' : 'cmp-adaptiveform-recaptcha';
```

Comment:
Why did we implement this way ? Ideally we should have two component types like we have done for core component, this way it is simple was customers to understand and also more discoverable.

Reply:
This was done after @amit-mnnit 's  request 
https://github.com/adobe/aem-forms-headless-components/pull/136#discussion_r1570045574

Reply:
I will rever this change.